src/gtest/test_foundersreward.cpp

```diff
@@ -275,9 +275,7 @@ TEST(FundingStreamsRewardTest, Zip207Distribution) {
 }
 
 TEST(FundingStreamsRewardTest, ParseFundingStream) {
```

Comment:
The tests that were being made here no longer made sense (they're effectively disallowed by construction) so they've been replaced by tests that _do_ make sense.